src/lib/trips.js

```diff
@@ -91,16 +91,24 @@ const getFeatureMode = feature => {
   return manualMode || sensedMode
 }
 
+const getFeatureModes = feature => {
+  return feature.features.map(feature => getFeatureMode(feature))
```

Comment:
not necessary return here

Reply:
*A débattre au prochain point, je ne pas pour ajouter ce genre de règle :)*